src/Utilities.jsx

```diff
@@ -0,0 +1,43 @@
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+const notifyError = (text) =>
+  toast.error(text, {
+    position: "top-left",
+    autoClose: 5000,
+    hideProgressBar: false,
+    closeOnClick: true,
+    pauseOnHover: true,
+    draggable: true,
+    progress: undefined,
+  });
+const notifySuccess = (text) =>
+  toast.success(text, {
+    position: "top-left",
+    autoClose: 5000,
+    hideProgressBar: false,
+    closeOnClick: true,
+    pauseOnHover: true,
+    draggable: true,
+    progress: undefined,
```

Comment:
this piece of code is repeating in every toast state, you can jut store this in a constant and use that.